docs/UserGuide.md

```diff
@@ -249,7 +249,8 @@ The following points explain the format of a command.
 ### Role Restrictions <a id="c-role-restrictions"></a>
 
 * The `ROLE_NAME` should only contain alphanumeric characters, spaces and an optional pair of round brackets.
-* The `REMINDER_DATE` should be in format dd-MM-yyyy HH:mm.
+* The `REMINDER_DATE` should not be in the past and must be a valid date 
```

Comment:
Nice job making it more clear to users that the reminder should not be in the past.